docs/core/whats-new/dotnet-8.md

```diff
@@ -520,16 +543,22 @@ The <xref:System.ComponentModel.DataAnnotations?displayProperty=fullName> namesp
 New APIs let you attach key-value pair tags to <xref:System.Diagnostics.Metrics.Meter> and <xref:System.Diagnostics.Metrics.Instrument> objects when you create them. Aggregators of published metric measurements can use the tags to differentiate the aggregated values.
 
 ```csharp
-MeterOptions options = new MeterOptions("name")
+var options = new MeterOptions("name")
 {
     Version = "version",
     // Attach these tags to the created meter
-    Tags = new TagList() { { "MeterKey1", "MeterValue1" }, { "MeterKey2", "MeterValue2" } }
+    Tags = new TagList()
```

Comment:
This could also use collection expressions *if* `TagList` is a collection type (has `IEnumerable` and an accessible `Add` method)